src/components/SecondaryPanes/Breakpoints/index.js

```diff
@@ -60,28 +44,28 @@ class Breakpoints extends Component<Props> {
 
     const isEmpty = breakpointSources.length == 0;
 
-    const exceptionsBox = createExceptionOption(
-      L10N.getStr("pauseOnExceptionsItem2"),
-      shouldPauseOnExceptions,
-      () => pauseOnExceptions(!shouldPauseOnExceptions, false),
-      "breakpoints-exceptions"
-    );
-
-    const ignoreCaughtBox = createExceptionOption(
-      L10N.getStr("pauseOnCaughtExceptionsItem"),
-      shouldPauseOnCaughtExceptions,
-      () => pauseOnExceptions(true, !shouldPauseOnCaughtExceptions),
-      "breakpoints-exceptions-caught"
-    );
-
     return (
       <div
         className={classnames("breakpoints-exceptions-options", {
           empty: isEmpty
         })}
       >
-        {exceptionsBox}
-        {shouldPauseOnExceptions ? ignoreCaughtBox : null}
+        <ExceptionOption
+          className="breakpoints-exceptions"
+          label={L10N.getStr("pauseOnExceptionsItem2")}
+          isChecked={shouldPauseOnExceptions}
+          onChange={() => pauseOnExceptions(!shouldPauseOnExceptions, false)}
+        />
+
+        <ExceptionOption
```

Comment:
I think a connected component would help us modularize a bit more here:

these props could all move inside of the component
```js
  shouldPauseOnExceptions: boolean,
  shouldPauseOnCaughtExceptions: boolean,
  pauseOnExceptions: Function
```

some day we might want to move this UI and it would be nice if it were self contained

Reply:
Do you want this done now, or make a component now and promote it to a container later?

Reply:
up to yo

Reply:
Let's do this the right way the first time to avoid more maintenance down the road.  This isn't a pressing task, though it is important.  Nice work so far @lukaszsobek !

Reply:
Hmm, on second glance, i think this is fine as is.